deploy/examples/toolbox-rook-image.yaml

```
@@ -0,0 +1,136 @@
+#################################################################################################################
+# Define the yaml settings for the Configure s5cmd
+
+# For example
+#   kubectl create -f toolbox-rook-image.yaml
+#################################################################################################################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-tools-s5cmd-configure
```

Since we could add other binaries in the future besides s5cmd, let's keep the generic name.
```suggestion
  name: rook-ceph-tools-rook-image
```